build_projects.yml

```diff
@@ -13,17 +14,19 @@ variables:
 
 jobs:
 - job: Projects
-  pool: LocalPool
+  pool: 
+    name: Default
+    demands:
+      - agent.name -equals no-OS
   steps:
   - checkout: self
     fetchDepth: 50
     submodules: true
     clean: true
-    persistCredentials: true
   - script: 'python3 ./tools/scripts/build_projects.py . 
                -export_dir $(RELEASE_DIR)
                -log_dir $(LOG_DIR)
-               -builds_dir $(BUILDS_DIR)'
+               -builds_dir /no-OS_builds/builds'
```

Comment:
we should modify the BUILDS_DIR value in the azure variables group instead of hardcoding the path.